bears/vcs/git/CISkipInspectBear.py

```diff
@@ -0,0 +1,63 @@
+import re
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class CISkipInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    def run(self, dependency_results, filename_patterns: list,
+            appveyor_ci: bool = False, **kwargs):
```

Comment:
put kwargs on a separate line -- no need for a trailing comma after kwargs, as no other args can follow it